qhub/cli/_init.py

```diff
@@ -0,0 +1,496 @@
+
+    # SSL CERTIFICATE
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] but recommended. If you want your Nebari domain to use a Let's Encrypt SSL certificate, "
+            "all we need is an email address from you.\n\n"
+        )
+    )
+    ssl_cert = questionary.confirm(
+        "Would you like to add a Let's Encrypt SSL certificate to your domain?",
+        default=False,
+        qmark=qmark,
+    ).ask()
+
+    if ssl_cert:
+        inputs.ssl_cert_email = questionary.text(
+            "Which email address should Let's Encrypt associate the certificate with?",
+            qmark=qmark,
+        ).ask()
+
+    # ADVANCED FEATURES
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] and includes advanced configuration changes. "
+            "⚠️ caution is advised!\n\n"
+        )
+    )
+    if questionary.confirm(
+        "Would you like to make advanced configuration changes?",
+        default=False,
+        qmark=qmark,
+    ).ask():
+
+        # NAMESPACE
+        inputs.namespace = questionary.text(
+            "What namespace would like to use?",
+            default=inputs.namespace,
+            qmark=qmark,
+        ).ask()
+
+        # TERRAFORM STATE
+        inputs.terraform_state = questionary.select(
+            "Where should the Terraform State be provisioned?",
+            choices=enum_to_list(TerraformStateEnum),
+            qmark=qmark,
+        ).ask()
+
+        # KUBERNETES VERSION
+        inputs.kubernetes_version = questionary.text(
+            "Which Kubernetes version would you like to use?",
+            qmark=qmark,
+        ).ask()
+
+    handle_init(inputs)
+
+    rich.print(
+        (
+            "Congratulations, you have generated the all important [purple]nebari-config.yaml[/purple] file 🎉\n\n"
+            "You can always edit your [purple]nebari-config.yaml[/purple] file by editing the file directly.\n"
+            "If you do make changes to it you can ensure its still a valid configuration by running:\n\n"
+            "\t\t[green]nebari validate --config path/to/nebari-config.yaml[/green]\n\n"
+        )
+    )
+
+    base_cmd = f"nebari init {inputs.cloud_provider}"
+
+    def if_used(key, model=inputs, ignore_list=["cloud_provider"]):
+        if key not in ignore_list:
+            b = "--{key} {value}"
+            value = getattr(model, key)
+            if isinstance(value, str) and (value != "" or value is not None):
+                return b.format(key=key, value=value).replace("_", "-")
+            if isinstance(value, bool) and value:
+                return b.format(key=key, value=value).replace("_", "-")
+
+    cmds = " ".join(
+        [_ for _ in [if_used(_) for _ in inputs.dict().keys()] if _ is not None]
+    )
+
+    rich.print(
+        (
+            "Here is the previous Guided Init if it was converted into a [green]nebari init[/green] command:\n\n"
```

Comment:
This seems mainly informational, and doesn't need any action from the user, right? In that case, it might be worth stating that explicitly. Based on our demo last week, I was almost about to run this before realizingwe already generated the `yaml` file. :)

```suggestion
            "For reference, if the previous Guided Init answers were converted into a direct [green]nebari init[/green] command, it would be:\n\n"
```